docs/src/user-guide/guides-using-object-storage/object-storage-config.md

```diff
@@ -0,0 +1,152 @@
+# Configuring object storage
+
+To use object storage with CLP, follow the steps below to configure the CLP IAM user and your object
+storage bucket(s) for each use case you require.
+
+## Configuration for compression
+
+[Attach the policy][add-iam-policy] below to the CLP IAM user (you can use the JSON editor),
+replacing the fields in angle brackets (`<>`) with the appropriate values:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "s3:GetObject",
+      "Resource": [
+        "arn:aws:s3:::<bucket-name>/<all-logs-prefix>*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": "s3:ListBucket",
+      "Resource": [
+        "arn:aws:s3:::<bucket-name>"
+      ],
+      "Condition": {
+        "StringLike": {
+          "s3:prefix": "<all-logs-prefix>*"
+        }
+      }
+    }
+  ]
+}
+```
+
+* `<bucket-name>` should be the name of the S3 bucket containing your logs.
+* `<all-logs-prefix>` should be the prefix of all logs you wish to compress.
+
+  :::{note}
+  If you want to enforce that only logs under a directory-like prefix, e.g., `logs/`, can be
+  compressed, you can append a trailing slash (`/`) after the `<all-logs-prefix>` value. This will
+  prevent CLP from compressing logs with prefixes like `logs-private`. However, note that to
+  compress all logs under the `logs/` prefix, you will need to include the trailing slash when
+  invoking `sbin/compress.sh` below.
+  :::
+
+## Configuration for archive storage
+
+[Attach the policy][add-iam-policy] below to the CLP IAM user (you can use the JSON editor),
+replacing the fields in angle brackets (`<>`) with the appropriate values:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+   {
+    "Effect": "Allow",
+    "Action": [
+      "s3:GetObject",
+      "s3:PutObject"
+    ],
+    "Resource": [
+      "arn:aws:s3:::<bucket-name>/<key-prefix>/*"
+    ]
+   }
+  ]
+}
+```
```

Comment:
The indentation is set to one space. let's be consistent with the other jsons which use two spaces.

```suggestion
```json
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": [
        "s3:GetObject",
        "s3:PutObject"
      ],
      "Resource": [
        "arn:aws:s3:::<bucket-name>/<key-prefix>/*"
      ]
    }
  ]
}
```
```